hypothesis-python/src/hypothesis/extra/ghostwriter.py

```diff
@@ -1051,7 +1072,7 @@ def test_fuzz_compile(pattern, flags):
     you'll need to specify a strategy - for example :func:`~hypothesis.strategies.text`
     or :func:`~hypothesis.strategies.binary`.  After that, you have a test!
     """
-    if not callable(func):
+    if not callable(func) or inspect.isclass(func):
```

Comment:
In these cases the existing behaviour is fine; if the class is callable people can ghostwrite tests for it.

Reply:
ok so you want to keep the class itself being testable, got it